ci-operator/step-registry/openshift-qe/cluster-health/openshift-qe-cluster-health-commands.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -eu
 
-oc version
-oc get node
-oc adm wait-for-stable-cluster --minimum-stable-period=${MINIMUM_STABLE_PERIOD} --timeout=${TIMEOUT}
+if [ ${PUBLIC_VLAN} == "true" ]; then
```

Comment:
We cannot merge this like this, the cluster-health step is super important, this way you are skipping it for private deployments.
What you need to do is something similar to what is being done in that PR, to make sure it will work both for public and private VLAN deployments: https://github.com/openshift/release/pull/68230